playbook.yml

```diff
@@ -227,3 +227,12 @@
       when: enable_mailpit
   tags:
     - eus
+
+
+- name: Yoda report
```

Comment:
We can move this role to 'Provision Rulesets on iCAT Server' and introduce a new variable, 'enable_yoda_report.' This will allow us to determine on a per-environment basis whether we want to install the reporting features.

Reply:
I have updated the playbook